src/ui/cuicui/application-ui/battery/battery-indicator/battery-indicator.tsx

```diff
@@ -25,9 +25,10 @@ export const BatteryIndicator: React.FC<BatteryInfoProps> = ({
   dischargingTime,
   className,
 }) => {
-  const batteryLevel = level ?? 0;
+  const batteryLevel = Math.round(level ?? 0); // Round the battery level
```

Comment:
Given the battery level, it would be beneficial to rename the variable to `roundedBatteryLevel` instead of adding a comment. This would eliminate unnecessary complexity in variable names and make the code clearer.

Reply:
I made the PR too quickly, I completely deleted this line!